packages/vapor/scss/tables/table.scss

```
@@ -191,6 +191,10 @@
         grid-template-columns: repeat(auto-fit, minmax(450px, 1fr));
         padding: $mod-card-list-vertical-padding $mod-card-list-horizontal-padding;
 
+        &:hidden {
```

I think it should be 
```suggestion
        &.hidden {
```